polyply/tests/test_top_parser.py

```diff
@@ -272,3 +293,21 @@ def test_buckingham_fail():
         top = Topology(force_field, name="test")
         with pytest.raises(IOError):
             polyply.src.top_parser.read_topology(new_lines, top)
+
+    @staticmethod
+    @pytest.mark.parametrize('lines', (
+        """
+        [cmaptypes]
+        something something
+        """,
+        """
+        [implicit_genborn_params]
+        something something
+        """
+         ))
+    def test_skip_directives(lines):
+        new_lines = textwrap.dedent(lines)
+        new_lines = new_lines.splitlines()
+        force_field = vermouth.forcefield.ForceField(name='test_ff')
+        top = Topology(force_field, name="test")
+        assert True
```

Comment:
ditto.
At least needs a comment that you're making sure there's no error, and why there shouldn't be.

Reply:
I added a doc-string. I think for now these tests test what they should. Also this is very obscure grompp behavior which I think is purely for legacy reasons.